### PR TITLE
6193 - Calendar: use abbreviated letter as default

### PR DESCRIPTION
**Pull request by edinfor**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

Current default is to use 1 letter to show day of the week, however, based on the IDS design pattern, looks like for calendar in page view, 3 letter should be the default. 

**Related github/jira issue (required)**:
Fixes https://github.com/infor-design/enterprise/issues/6193

**Steps necessary to review your pull request (required)**:
- compare http://localhost:4000/components/calendar/test-range-in-month.html  -> https://main-enterprise.demo.design.infor.com/calendar/test-range-in-month.html 

**Included in this Pull Request**:
- [ ] An e2e or functional test for the bug or feature.
- [x] A note to the change log.

### CODE REVIEW

**Review by tmcconechy — APPROVED** [review]
fyi dont worry about the build error im fixing - but we may need to update an image when its fixed

**Comment by tmcconechy** [comment]
@edinfor please merge with main - the tests should be fixed to run again

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@
 
 - `[Homepage]` Fixed instability of the visual tests. ([#6179](https://github.com/infor-design/enterprise/issues/6179))
 - `[Widget]` Fix on drag image including the overflow area. ([NG#1216](https://github.com/infor-design/enterprise-ng/issues/1216))
+- `[Calendar]` Fix day of the week to show three letters as default in range calendar
```
Comment (ericangeles) [inline docs/CHANGELOG.md:9]: @edinfor can you update the changelog by adding the ticket issue number and the link of it? And make alphabetical order. Thanks!

**Comment by edinfor** [comment]
@tmcconechy this is the range calendar that needs to be fixedhttp://localhost:4000/components/calendar/test-range-in-month.html

**Review by tmcconechy — APPROVED** [review]

**Review by tjamesallen15 — APPROVED** [review]

**Review by ericangeles — APPROVED** [review]

**Pull request merged**